.github/workflows/test-coverage.yml

```diff
@@ -236,6 +236,27 @@ jobs:
           coverage-artifact-name: "coverage_encryption"
           coverage-path: coverage.txt
 
+  # This job runs the database with telemetry tests using default configuration, on linux.
+  test-telemetry:
+    name: Test telemetry job
+
+    runs-on: ubuntu-latest
+
+    env:
+      GO_FLAGS: -tags=telemetry
```

Comment:
question: is `GO_FLAGS=-tags=telemetry` and `GOFLAGS=-tags=telemetry` the same?

Reply:
Turns out the CI was just flaky. Coverage now shows that it is working.

Reply:
It wasn't a suggestion, but a question haha because I only saw that flag in documentation and not the one with underscore. Cool so it works eh?

Reply:
The configure telemetry function in otel.go file is untested? Is that expected?

Reply:
I've added a simple smoke test